client/layout/guided-tours/tours/jetpack-lazy-images-tour/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import meta from './meta';
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Quit,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+export const JetpackLazyImagesTour = makeTour(
+	<Tour { ...meta }>
+		<Step
+			name="init"
+			target=".jetpack-lazy-images-settings .form-toggle__switch"
+			arrow="top-left"
+			placement="below"
+			style={ {
+				animationDelay: '0.7s',
+				zIndex: 1,
+			} }
+		>
+			{ ( { translate } ) => (
+				<Fragment>
+					<p>{ translate( "Let's boost load times for your site by lazy-loading images." ) }</p>
```

Comment:
Let's speed up your page load times with lazy-loading images -- images that only load when a visitor  scrolls down to see them.

(I feel like we should explain, for people who aren't otherwise familiar/haven't read other parts of the UI? But if it's too much text, we can adjust.)